.goreleaser.yaml

```diff
@@ -0,0 +1,2 @@
+builds:
+- skip: true
```

Comment:
wait what does this do

Reply:
This allows us to run goreleaser without a main